### DISCUSSION TITLE
Calendar/Date Field

### DISCUSSION THREAD

**Discussion by luismgarcia** [tobimori/kirby-dreamform#18] [Feature Requests]
Field to select date

**Comment by tillschander** [comment]
Hey everyone! I was in need of a date field as well. So I've build one. And now I’m releasing it. Maybe it is of use to you too:

https://github.com/tillschander/dreamform-datefield

**Comment by tobimori** [comment]
I released the first official version of a date picker field:
https://github.com/tobimori/dreamform-date-field

This is a separate plugin, since it uses a third-party library and requires JavaScript on the frontend. I will close this, since they are now two solutions for those wanting a Date Field in DreamForm.
